tests/integration/security/reachability_test.go

```diff
@@ -91,6 +91,45 @@ func TestReachability(t *testing.T) {
 						return opts.PortName != "http"
 					},
 				},
+				{
+					ConfigFile:          "beta-mtls-on.yaml",
```

Comment:
would it help to have test cases where we have both beta and alpha policy? that can be a typical case during migration.

and what's the intended behavior? to me, it seems like if there's one beta policy instance (ns/mesh/workload) can be applied to a particular workload, all alpha policy is ignored, is that correct?

feel free to do it in follow-up PR and leave a TODO.

Reply:
It's already there.

Reply:
which one? I didn't see it in the e2e test. all config file only references the beta policy.